### PR TITLE
Chore/accessibility audit as

### PR DESCRIPTION
**Pull request by AsherSpurr**
# Pull Request
Accessibility audit
## Description
Tested with Lighthouse, WAVE, Tabbing, ScreenReader + made appropriate changes.
## Changes Made
I wish I could fully remember all the little changes.
- Place tab-index on cards to make them focusable
- ^^Add keypress event to interactive elements/card
- Add aria-label and roles ONLY to things that NEEDED it. I.E. the cards that are interactive but show no sign via screen reader.
- change the favorite button to SVG -> refactor associated code
- ^^ Add conditional rendering in Card for heart SVG vs Remove SVG
- Update a ton of labeling
- add CSS to the education page
- Remove a few lines from the Home page
## Checklist
- [x] The code follows the project's coding standards.
- [ ] Unit tests have been added or updated to cover the changes.
- [x] Documentation has been updated to reflect the changes (if applicable).
- [x] The code compiles without errors.
- [ ] The changes have been tested locally and pass all relevant tests.
- [ ] All new and existing tests pass.
- [x] The pull request has been reviewed by at least one other contributor.

## Reviewer Instructions
Looking through the commits will be more helpful than this breakdown.

### CODE REVIEW

**Comment by corysanders3** [comment]
It looks like this has merge conflicts. Do we want to review them during our meeting this evening?

**Comment by tednaphil** [comment]
> It looks like this has merge conflicts. Do we want to review them during our meeting this evening?

I think reviewing all together sounds good! We could also use this time to discuss remaining items to test.

File: src/components/teaArticle/TeaArticle.tsx [hunk 1]
```diff
@@ -1,29 +1,50 @@
...
-      <p>{tea.oxidation}</p>
-      <p>{tea.heating}</p>
-      <p>{tea.climate}</p>
-      <button className="infoButton">
-        <NavLink to='/'>⃪ Go back</NavLink>
-      </button>
-      <button className="infoButton">
-        <NavLink to={`/tea/${slug}`}>See {slug} tea →</NavLink>
-      </button>
+    <div className='container'>
```
Comment (tednaphil) [inline src/components/teaArticle/TeaArticle.tsx:12]: The tea ed articles look so good!

**Review by corysanders3 — APPROVED** [review]
Nice looking code updates. Glad we cleared up the merge conflicts.

**Review by tednaphil — APPROVED** [review]
The styles look fabulous! Reminder for us to update the conditional to have the active fav heart persist

**Pull request merged**